staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -382,6 +415,24 @@ func setPrivileged(p *corev1.Pod, containerName string) {
 	})
 }
 
+// setPrivilegedWindowsOS configures the Windows containers as Host Process Container on Windows Server.
+func setPrivilegedWindowsOnOS(p *corev1.Pod, containerName string) {
+	podutils.VisitContainers(&p.Spec, podutils.AllContainers, func(c *corev1.Container, _ podutils.ContainerType) bool {
+		if c.Name != containerName {
+			return true
+		}
```

Comment:
Where does this pod spec come from? All the containers in a HPC pod must be configured with `HostProcess`, so If this pod spec has more than one container I think this wouldn't work.

Since this only used with `kubectl debug node command` could this just set the `Hostprocess` setting on the pod spec? and verify it isn't false already on any of the containers?